### PR TITLE
Creating workflows

### PR DESCRIPTION
**Pull request by Albertoimpl**

### CODE REVIEW

**Review by fifthposition — APPROVED** [review]

File: README.adoc [hunk 1]
```diff
@@ -1,2 +1,120 @@
 = Spring Cloud App Broker Samples
 
+
+== Parameters transformers
+
+One of the very first things every broker will need to do is handling user input parameters.
```
Comment (royclarkson) [inline README.adoc]: handling -> handle

File: README.adoc [hunk 2]
```diff
@@ -1,2 +1,120 @@
...
+----
+@Bean
+public ParametersTransformerFactory<BackingApplication, Object> requestTimeoutParameterTransformerFactory() {
+        return new RequestTimeoutParameterTransformer();
+}
+----
+
+== Creating workflows
+
+With the default configuration, `spring-cloud-app-broker` handles the implementation of the basic operations a broker can handle: creating, updating, deleting, binding and unbinding.
```
Comment (royclarkson) [inline README.adoc]: "create, update, delete, bind, and unbind" would read a bit better

File: creating-workflows/src/main/java/org/springframework/cloud/appbroker/samples/creatingworkflows/AppBrokerApplication.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.appbroker.samples.creatingworkflows;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SuppressWarnings({"PMD.UseUtilityClass"})
```
Comment (royclarkson) [inline creating-workflows/src/main/java/org/springframework/cloud/appbroker/samples/creatingworkflows/AppBrokerApplication.java:22]: hmm. that's annoying to have to add this.

**Review by royclarkson — COMMENTED** [review]
👍 added some minor comments

File: creating-workflows/src/main/java/org/springframework/cloud/appbroker/samples/creatingworkflows/AppBrokerApplication.java (hunk 3 above)
Comment (Albertoimpl) [reply creating-workflows/src/main/java/org/springframework/cloud/appbroker/samples/creatingworkflows/AppBrokerApplication.java:22]: The alternative was to ignore this rule. I don't have strong feelings either way.

**Comment by Albertoimpl** [comment]
Thanks for the feedback!

**Pull request merged**